# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/gtk/WaylandSurface.h
# Commit: 3927f21c92c9
# Full Hash: 3927f21c92c941c9efd0b00b0dc4cfc2b905452f
# Author: stransky <stransky@redhat.com>
# Date: 2025-09-08 08:22:14
# Description:
#   Bug 1979106 [Wayland] Don't recycle buffer transactions across WaylandSurfaces r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D263545
# ==============================================================================

diff -r 2a24e5601893 -r 3927f21c92c9 widget/gtk/WaylandSurface.h
--- a/widget/gtk/WaylandSurface.h	Mon Sep 08 02:52:34 2025 +0000
+++ b/widget/gtk/WaylandSurface.h	Mon Sep 08 04:20:40 2025 +0000
@@ -300,6 +300,7 @@
   void Commit(WaylandSurfaceLock* aProofOfLock, bool aForceCommit,
               bool aForceDisplayFlush);
 
+  // Get buffer transaction for WaylandBuffer, create new or recycle one.
   BufferTransaction* GetNextTransactionLocked(
       const WaylandSurfaceLock& aSurfaceLock, WaylandBuffer* aBuffer);
   // Force release/detele all transactions and wl_buffers attached to them.
